docs/api/covidcast_clients.md

```diff
@@ -6,15 +6,15 @@ nav_order: 1
 
 # COVIDcast Libraries
 
-Dedicated COVIDcast access and analysis libraries are available for several languages:
+To access COVIDcast data, we recommend using the following client libraries:
 
-* R: [covidcast](https://cmu-delphi.github.io/covidcast/covidcastR/)
-* Python: [covidcast](https://cmu-delphi.github.io/covidcast/covidcast-py/html/)
+- R: [epidatr](https://cmu-delphi.github.io/epidatr/),
+- Python: [covidcast](https://cmu-delphi.github.io/covidcast/covidcast-py/html/) (soon to be replaced [epidatpy](https://github.com/cmu-delphi/epidatpy)).
 
 These packages provide a convenient way to obtain COVIDcast data as a data frame
-ready to be used in further analyses and provide convenient mapping and
-analysis functions. For installation instructions and examples, consult their
-respective webpages.
+ready to be used in further analyses and provide convenient mapping and analysis
+functions. For installation instructions and examples, consult their respective
+webpages.
 
-
-To access Epidata datasets on other diseases, visit our [Epidata API Client Libraries)](client_libraries.md) documentation.
+To access Epidata datasets on other diseases, visit our
+[Epidata API Client Libraries](client_libraries.md) documentation.
```

Comment:
I put this link back on the same line to get it to work.

Reply:
Gotcha, I think the issue is the unmatched ) parenthesis, but your change fixed that too 👍